documentation/modules/ROOT/pages/connectors/mongodb-sink.adoc

```diff
@@ -0,0 +1,375 @@
+[[mongodb-sink-examples]]
+== Example configuration
+
+[source,json,indent=0,subs="+attributes"]
+----
+
+{
+    "name": "mongodb-sink-connector",
+    "config": {
+        "connector.class": "io.debezium.connector.mongodb.sink.MongoDbSinkConnector",
+        "topics": "dbserver1.inventory.customers,dbserver1.inventory.orders,dbserver1.inventory.products",
+        "mongodb.connection.string": "mongodb://localhost:27017",
+        "sink.database": "debezium"
+    }
+}
+----
+
+
+// Type: concept
+// Title: Monitoring the {prodname} MongoDB sink connector
+// ModuleID: monitoring-of-debezium-mongodb-sink-connector
+[[mongodb-sink-monitoring]]
+== Monitoring
+
+This release of the connector does not expose any metrics.
+
+
+// Type: concept
+// Title: Key field mapping of the {prodname} MongoDB sink connector
+// ModuleID: key-field-mapping-of-debezium-mongodb-sink-connector
+[[mongodb-sink-key-field-mapping]]
+== Key field mapping
+
+* Keys from {prodname} change events, such as Kafka message keys, are mapped to the MongoDB `_id` field by default.
+* Values are mapped into MongoDB documents.
+* Updates and deletes are resolved based on the key field mapping.
+
+The following example shows an event key in a Kafka topic:
+[source,json,indent=0,subs="+attributes"]
+----
+{
+    "userId": 1,
+    "orderId": 1
+}
+----
+
+Based on the mapping logic, the preceding key is mapped to the `_id` field in a MongoDB document, as showin in the following example:
+[source,json,indent=0,subs="+attributes"]
+----
+{
+    "_id": {
+        "userId": 1,
+        "orderId": 1
+    }
+}
+----
+
+// Type: concept
+// Title: Using CloudEvents with the {prodname} MongoDB sink connector
+// ModuleID: cloudevents-with-debezium-mongodb-sink-connector
+[[mongodb-sink-cloudevents]]
+== Using CloudEvents with {prodname} MongoDB Sink Connector
+
+The {prodname} MongoDB sink connector can consume records serialized as CloudEvents.
+{prodname} can emit change events in CloudEvents format, so that the event payload is encapsulated in a standardized envelope.
+
+When you enable CloudEvents on the source connector, the MongoDB sink connector parses the CloudEvents envelope.
+
+The actual {prodname} event payload is extracted from the data section.
+
+The event is then applied to the target MongoDB collection, following the standard insert, update, or delete semantics.
+
+This process makes it possible to integrate {prodname} with broader event-driven systems while still persisting the resulting events in MongoDB.
+
+.CloudEvents sink options
+[cols="30%a,25%a,45%a"]
+|===
+|Property | Default | Description
+
+|[[mongodb-sink-property-cloud-events]]<<mongodb-sink-property-cloud-events, `+cloud.events.schema.name.pattern+`>>
+| `.*CloudEvents\.Envelope$`
+| Regular expression pattern to identify CloudEvents messages by matching the schema name with this pattern.
+
+|===
+
+
+// Type: concept
+// Title: Next steps coming for {prodname} MongoDB Sink Connector
```

Comment:
```suggestion
// Title: Next steps coming for {prodname} MongoDB sink connector
```

Reply:
I'm not clear on the purpose of this topic, and I wonder whether we should remove it or conditionalize it.
Is the materialization of relational change events something that is currently available in the community version? Or is this forward-looking?


If this content is relevant now across editions, I think that rather than insert this content here at the end, we should refer to it in the overview section.

Reply:
This is all forward-looking. Presently, the sink connector only allows for the serialization of Relational Connector events into a MongoDB target; however, in the future, we want to support MongoDB sources as well. 

That said, the first bullet seems inverted using "into a MongoDB" rather than "from a MongoDB". For the second bullet, I don't have sufficient context to understand exactly what is being implied here, but perhaps its related to the column/data type propagation details we discuss in the JDBC sink.

In either case, I think as long as we have admonitions that outline the current limitations, which if I recall we do already, then I think we can safely remove this section.  wdyt @roldanbob ?